XamarinCommunityToolkit/Views/GravatarImageSource/iOS/GravatarImageSourceHandler.ios.macos.cs

```diff
@@ -0,0 +1,18 @@
+using Foundation;
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+	public partial class GravatarImageSourceHandler
+	{
+		static string GetCacheDirectory()
+		{
+			var dirs = NSSearchPath.GetDirectories(NSSearchPathDirectory.CachesDirectory, NSSearchPathDomain.User);
+			if (dirs == null || dirs.Length == 0)
+			{
+				// this should never happen...
```

Comment:
Do you mind to add a log here, telling the dev that something odd happened? Just to make easy for him to track the error and report